internal/dockercompose/state.go

```diff
@@ -101,6 +104,34 @@ func (s State) HasEverBeenReadyOrSucceeded() bool {
 	return !s.LastReadyTime.IsZero()
 }
 
+func populateContainerStatus(state *types.ContainerState) string {
+	if state.Health == nil {
+		return state.Status
+	}
+	if state.Health.Status == types.Starting {
+		return ContainerStatusRestarting
+	}
+	return state.Status
+}
+
+func populateContainerError(state *types.ContainerState) string {
+	if state.Error != "" {
+		return state.Error
+	}
+	if state.Health == nil || state.Health.Status != types.Unhealthy {
+		return ""
+	}
+	result := "container is unhealthy"
```

Comment:
is this log displayed anywhere today? i didn't see it anywhere in the UI, only in the tilt cli output

Reply:
I didn't find it on the UI either.